src/docs/sdk/envelopes.mdx

```diff
@@ -255,7 +255,9 @@ EOF:**
 Each Envelope consists of headers and a potentially empty list of Items, each
 with their own headers. Which Headers are required depends on the Items in an
 Envelope. This section describes all Item types and their respective required
-headers.
+headers. It is worth noting that the list of Item types doesn't match the data
```

Comment:
```suggestion
headers. It is worth noting that the list of Item types don't match the data
```

`don't` since list of item types is plural, I guess?!

Reply:
Hmm, I'm referring to the list here, which is singular and not plural. I think singular should be fine.